app/javascript/components/visualization/controls/ScatterPlotLegend.js

```diff
@@ -203,6 +290,32 @@ export default function ScatterPlotLegend({
           </>
           }
         </div>
+        { enableColorPicking &&
+          <div>
+            { showColorControls &&
+              <>
+                <span>Click a label to select a new color</span><br/>
+                <div>
+                  <a role="button" data-analytics-name="legend-color-picking-save" onClick={saveColors}>
+                    Save colors <FontAwesomeIcon icon={faSave}/>
+                  </a>
+                  <a role="button" className="pull-right" data-analytics-name="legend-color-picking-save" onClick={cancelColors}>
+                    Cancel
+                  </a><br/>
+                  &nbsp;
+                  <a role="button" className="pull-right" data-analytics-name="legend-color-picking-save" onClick={resetColors}>
```

Comment:
Blocker: `reset`, not `save`.

```suggestion
                  <a role="button" className="pull-right" data-analytics-name="legend-color-picking-reset" onClick={resetColors}>
```